docs/framework/svelte/reference/functions/createInfiniteQuery.md

```
@@ -3,44 +3,134 @@ id: createInfiniteQuery
+
+[`Accessor`](../type-aliases/Accessor.md)\<[`DefinedInitialDataInfiniteOptions`](../type-aliases/DefinedInitialDataInfiniteOptions.md)\<`TQueryFnData`, `TError`, `TData`, `TQueryKey`, `TPageParam`\>\>
+
+#### queryClient?
+
+[`Accessor`](../type-aliases/Accessor.md)\<`QueryClient`\>
+
+### Returns
+
+[`DefinedCreateInfiniteQueryResult`](../type-aliases/DefinedCreateInfiniteQueryResult.md)\<`TData`, `TError`\>
+
+## Call Signature
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)